docs/api.rst

```diff
@@ -348,6 +355,41 @@ Note that ``->>`` is used to compare ``blood_type`` as ``text``. To compare with
     { "id": 15, "age": 35 }
   ]
 
+The arrow operators are also used for array and composite type columns.
```

Comment:
How about a new linkable section for these? 

```suggestion
Composite / Array Columns
-------------------------------------

The arrow operators are also used for array and composite type columns.
```

That way we can showcase the feature better in the release notes.

The above `Array / Composite / JSON Columns` would have to be changed to just `JSON columns`.